SPECS/azure-iot-sdk-c/azure-iot-sdk-c.spec

```diff
@@ -86,6 +90,9 @@ install -p -m 755 provisioning_client/tools/tpm_device_provision/tpm_device_prov
 /usr/cmake/*
 
 %changelog
+*   Thu Jan 04 2024 Karim Eldegwy <karimeldegwy@microsoft.com> - 2023.08.07-1
+-   Upgrate to  2023.08.07-1
```

Comment:
No need to specify the release in this comment - just the version. Also an extra space is present. Suggest:
`Upgrade to 2023.08.07`